docs/metrics.md

```diff
@@ -172,15 +172,11 @@ the duration of a put in the wait queue
 how many rows come per get response
 * `store.cassandra.to_iter`:  
 the duration of converting chunks to iterators
-* `tank.add_to_saved_chunk`:  
-points received - by a secondary node - for the most recent chunk when that chunk
-has already been saved by a primary.  A secondary can add this data to its chunks.
-* `tank.add_to_saving_chunk`:  
-points received - by the primary node - for the most recent chunk
-when that chunk is already being saved (or has been saved).
-this indicates that your GC is actively sealing chunks and saving them before you have the chance to send
-your (infrequent) updates.  The primary won't add them to its in-memory chunks, but secondaries will
-(because they are never in "saving" state for them), see below.
+* `tank.add_to_closed_chunk`:    
+points received for the most recent chunk when that chunk is already being "closed",
+ie the end-of-stream marker has been written to the chunk.
+This indicates that your GC is actively sealing chunks and saving them before you have the chance to send
+your (infrequent) updates.  Any points revcieved for a chunk that has already been closed are discarded.
```

Comment:
It is possible. consider this:
* we're adding points t1, t2, t3, t4, t5 (all have the same t0 i.e. go into the same chunkspan)
* there's a long time between t2 and t3, let's say it's a time just under chunk-max-stale.
* A is primary, B is secondary.
* A consumes t1, t2, and then experiences some lagging. t3 is in kafka but A doesn't consume it quite yet
* A's GC notices that last write was before chunkMinTs, closes the chunk and persists it.
* A then consumes the points t3 t4 and t5 but rejects them. This is worth instrumenting (it indicates that maybe you should check out your GC parameters).  it does increment `add_to_closed_chunk` so you can see it on your dashboard when looking at the instance's stats.

* B on the other hand, does not experience the lagging.
* it consumes t1, t2 but does not experience the same lag as A. it consumes t3.
* B's GC sees a last write more recent than chunkMinTs, so doesn't close the chunk.
* B receives notification from A and updates lastSaveStart and lastSaveFinish accordingly, but does not close any chunk.
* B consumes t4 and t5, and can push them to the chunk, as it is not closed and we don't look at the lastSave* timestamps.

=> B now added t3, t4 and t5 to its chunk, but we know A did not (we don't know for t3 but we do know for t4 and t5)

there's two problems:

* when loading your charts, datasource B will show more points than datasource A (when loading from memory)
* datasource B will show less points once it needs to retrieve the chunk from cassandra

 
as B could foresee these problems happening (e.g. if it would check `lastSave*` when adding new points, just like the old code checked `currentChunk.Saved` when adding new points), it could inform the operator by incrementing this stat.